docs/site/tutorials/todo-list/todo-list-tutorial-relations.md

```diff
@@ -0,0 +1,141 @@
+---
+lang: en
+title: 'Add Model Relations'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/todo-list-tutorial-relations.html
+summary: LoopBack 4 TodoList Application Tutorial - Add TodoList Repository
+---
+
+### Define the model relation
+
+We are going to add the model relation to indicate the relation that `TodoList`
+_hasMany_ `Todo` using the `lb4 relation` command.
+
+```sh
+$ lb4 relation
+? Please select the relation type hasMany
+? Please select source model TodoList
+? Please select target model Todo
+? Foreign key name to define on the target model todoListId
+? Source property name for the relation getter todos
+? Allow TodoList queries to include data from related Todo instances? Yes
+   create src/controllers/todo-list-todo.controller.ts
+
```

Comment:
I'd like to add some explanations about the default foreign key name/source property names.

```
Notice: we use **default** foreign key and source property names in this case. 
If you'd like to customize them, please check `Relation Metadata` https://loopback.io/doc/en/lb4/HasMany-relation.html#relation-metadata 
and other relations as well.
```